lib/rules/no-html-safe.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const ERROR_MESSAGE = 'Do not use htmlSafe.';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow the use of htmlSafe',
+      category: 'Miscellaneous',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-html-safe.md',
+    },
+    fixable: null,
+    schema: [],
+  },
+
+  ERROR_MESSAGE,
+
+  create(context) {
+    return {
+      ImportDeclaration(node) {
+        if (
+          (node.source.value === '@ember/string' || node.source.value === '@ember/template') &&
+          node.specifiers.some((s) => s.local.name === 'htmlSafe')
+        ) {
+          context.report(node, ERROR_MESSAGE);
```

Comment:
Need to report the specific specifier `htmlSafe` instead of the entire import statement. We don't want to flag other specifiers that might be imported in the same statement.